### ISSUE TITLE
How was RichTextEditor?

### ISSUE THREAD

**Issue by atmoio** [tectiv3/amanothi#1]
I'm curious how the RichTextEditor performed in your tests? It looks like it's a webview wrapper. Did it feel native?

**Comment by tectiv3** [comment]
Yeah, it totally did! No problem with it whatsoever.

**Comment by atmoio** [comment]
Awesome. Glad I found it. Won't be using it right away but would make a great update in the future. I was searching "React Native notes app" and found this repo :) Was trying to see how others got over the annoying lack of scroll flinging in TextInput on Android. Looks like there's no way around it. So I've just made my own native EditText component.

**Comment by tectiv3** [comment]
Yeah, I was hoping to write a notes app myself when I was in between jobs.

Why are you moving to react native?

**Comment by atmoio** [comment]
> Why are you moving to react native?

Maintaining both an iOS and Android codebase is making it really difficult to add nice things to the app, or even maintain it at all. So I'm transitioning the official app to RN. 

https://github.com/standardnotes/sn-rn
